scripts/sct_label_utils.py

```diff
@@ -69,6 +69,14 @@ def __init__(self, fname_label, fname_output=None, fname_ref=None, cross_radius=
                 self.fname_output = fname_output
         else:
             self.fname_output = fname_output
+
+        if isinstance(fname_previous, list):
```

Comment:
why the need for this if? how about simply enforcing `fname_previous` to be a file name (could be verified by the parser)

Reply:
I think you are right it will be enough. I just modeled it after the fname_output  to begin with.